UnitySDK/Assets/ML-Agents/Scripts/LearningBrain.cs

```diff
@@ -1,9 +1,13 @@
-﻿using System;
+﻿#define ENABLE_BARRACUDA
```

Comment:
We should remove TF# and these define symbols before merging

Reply:
We will leave the TF# code in place for the next release in order to transition.